src/arfsdao.ts

```diff
@@ -718,6 +726,85 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 		return latestRevisionsOnly ? allFiles.filter(latestRevisionFilter) : allFiles;
 	}
 
+	async getEntitiesWithParentFolderId(
+		folderID: FolderID,
+		builder: (
+			node: GQLNodeInterface,
+			entityType: 'file' | 'folder'
+		) => ArFSFileOrFolderBuilder<ArFSFileOrFolderEntity>,
+		latestRevisionsOnly = true
+	): Promise<ArFSFileOrFolderEntity[]> {
+		let cursor = '';
+		let hasNextPage = true;
+		const allEntities: ArFSFileOrFolderEntity[] = [];
+
+		// TODO: Always derive owner of a wallet from the earliest drive a wallet by default
```

Comment:
Nit: Grammar
Also, we're going to make this the default behavior but at a later point will add the ability to NOT filter that way in order to find problematic (e.g. fraudulent or corrupted) entities.